### ISSUE TITLE
Doesn't compile with GHC 7.8.3 (roles)

### ISSUE THREAD

**Issue by Fuuzetsu** [lambdabot/lambdabot#84]
```
src/Lambdabot/Monad.hs-boot:4:1:
    Type constructor ‘LB’ has conflicting definitions in the module
    and its hs-boot file
    Main module: type role LB nominal
                 newtype LB a
                   = LB {runLB :: ReaderT (IRCRState, IORef IRCRWState) IO a}
    Boot file:   data LB a
```

http://hydra.cryp.to/build/164390/log/raw

**Comment by ocramz** [comment]
I can confirm, doesn't build on GHCi 7.8.3., due to the same type mismatch

**Comment by int-e** [comment]
This was fixed by #78, but has not been released to hackage yet.

**Comment by int-e** [comment]
The fix is now on hackage.
